### DISCUSSION TITLE
Global setting for git segment's display_status

### DISCUSSION THREAD

**Discussion by cesarcoatl** [JanDeDobbeleer/oh-my-posh#795] [Ideas]
Here I propose a way to get Git status back to life with a global setting, perhaps something could be set in our `$PROFILE`.

Unless there is already a way to accomplish this and I am not aware of it.

Thanks and great job!

**Comment by JanDeDobbeleer** [comment]
@thecesrom can I first ask why you're not [creating a custom theme](https://ohmyposh.dev/docs/pwsh#override-the-theme-settings) with `"display_status": true`? It's not "dead", the default sample themes simply don't have it enabled

**Reply by cesarcoatl** [reply]
I have not ventured in creating my own custom theme, but I have overridden the `git` block on the themes I use.

I was just wondering if there could be a way to enable the status without having to edit the theme file with each update.

Thanks again.

**Reply by JanDeDobbeleer** [reply]
@thecesrom are you editing the theme directly in the powershell module folder? Because that's not a good idea as it will indeed be overridden every time the module updates. You can copy that theme to your user home folder for example and keep it (we stay compatible as much as we can). Suppose you copy that theme json to `~/thecesrom.omp.json` you can make the module it with `Set-Poshprompt -Theme ~/thecesrom.omp.json`.
